packages/markdownviewer/src/widget.ts

```diff
@@ -48,7 +48,10 @@ class MarkdownViewer extends Widget {
   constructor(context: DocumentRegistry.Context, rendermime: RenderMime) {
     super();
     this.addClass(MD_CLASS);
-    this.layout = new PanelLayout();
+    let layout = this.layout = new PanelLayout();
+    let toolbar = new Widget();
```

Comment:
Why do the other things use a `::before` element, but this uses a `Widget` for the "toolbar"?